### PR TITLE
B-Reminders

### PR DESCRIPTION
**Pull request by sinteary**
Add reminders for upcoming tasks. By default "remind" will list tasks within 7 days from current time. User can also insert date, for example "remind 21/09/2019 2359" to be reminded of all tasks up until a certain time.

### CODE REVIEW

**Comment by iamabhishek98** [comment]
test approval

File: src/main/java/Duke.java [hunk 1]
```diff
@@ -1,6 +1,9 @@
 import exceptions.DukeException;
 import views.CLIView;
 
+/**
+ * Main class.
```
Comment (Lucria) [inline src/main/java/Duke.java]: @sinteary please remove comments at the bottom of the class.

File: src/main/java/controllers/ConsoleInputController.java [hunk 2]
```diff
@@ -18,7 +19,7 @@
     private CLIView consoleView;
     private TaskFactory taskFactory;
     private TaskList taskList;
-    private String filePath = "D:\\For Git\\CS2113_Duke\\src\\main\\java\\saves\\TaskLists.txt";
+    private String filePath = "src/main/saves/savefile.txt";
```
Comment (Lucria) [inline src/main/java/controllers/ConsoleInputController.java]: Please do not commit the savefile.txt into Git. Remove it with the next commit. Will cause errors

**Comment by Lucria** [comment]
@sinteary Resolve those stuff thanks

**Pull request merged**